### ISSUE TITLE
Index out of range in get_system_info_resp_parser for ISO15693

### ISSUE THREAD

**Issue by cryptax** [gvinet/pynfcreader#2]
Hi,
I am using your reader on a TagIt ISO 15693. It seems that, in my case, the Get System Info returns a response which is shorter than what you expect (only 12 bytes where you seem to be expecting 15). Consequently, the program crashes on index out of range.

```
INFO  ::  Get System Info
INFO  ::  Command:
INFO  ::  		22 2B A2 5F 7A 01 00 A0   07 E0                        "+._z...   ..
INFO  ::  	
INFO  ::  Response:
INFO  ::  		00 04 A2 5F 7A 01 00 A0   07 E0 F3 07 

...

pynfcreader/pynfcreader/sessions/iso15693.pyc in get_system_info_resp_parser(self, resp)
    119         parsing["AFI"]      = utils.int_array_to_hex_str(resp[11:12])
    120         mem_size = resp[12:14]
--> 121         block_size = (mem_size[1] & 0x1F) + 1
    122         parsing["MEM_SIZE_BLOCK_SIZE"] = block_size
    123         self._block_num  = mem_size[0] + 1

IndexError: list index out of range
```

Indeed, my response is only 12 bytes, so `resp[12:14]` won't provide anything, and `mem_size[1]` will fail.

**Comment by cryptax** [comment]
More information: in my case, my tag does not support AFI nor DSFID. It seems I automatically get after the UID the number of blocks. In my case, it is on 1 byte, and according to Tag Info app, blocks are of 8 bytes. So I have 244 blocks with 8 bytes per block = 1952 bytes.

This is a very quick and dirty fix in my case (but I don't think it is good to do it that way + I get also error on security lock):

```
diff --git a/pynfcreader/sessions/iso15693.py b/pynfcreader/sessions/iso15693.py
index c940c9f..2fc7dfe 100644
--- a/pynfcreader/sessions/iso15693.py
+++ b/pynfcreader/sessions/iso15693.py
@@ -115,17 +115,24 @@ class Iso15693Session(object):
     def get_system_info_resp_parser(self, resp):
         parsing = {}
         parsing["UID"]      = utils.int_array_to_hex_str(list(reversed(resp[2:1
0])))
-        parsing["DSFID"]    = utils.int_array_to_hex_str(resp[10:11])
-        parsing["AFI"]      = utils.int_array_to_hex_str(resp[11:12])
-        mem_size = resp[12:14]
-        block_size = (mem_size[1] & 0x1F) + 1
+        #parsing["DSFID"]    = utils.int_array_to_hex_str(resp[10:11])
+        #parsing["AFI"]      = utils.int_array_to_hex_str(resp[11:12])
+        parsing["DSFID"]    = utils.int_array_to_hex_str([00])
+        parsing["AFI"]      = utils.int_array_to_hex_str([00])
+        #mem_size = resp[12:14]
+        mem_size = resp[10:11]
+        # block_size = (mem_size[1] & 0x1F) + 1
+        block_size = 8
         parsing["MEM_SIZE_BLOCK_SIZE"] = block_size
         self._block_num  = mem_size[0] + 1
 
         parsing["MEM_SIZE_BLOCK_NB"] = self._block_num
         full_mem_size = block_size * self._block_num
-        parsing["MEM_SIZE_PRETTY_PRINT"] = "%d bytes : %d blocks of %d bytes (%s)" % (full_mem_size, self._block_num, block_size, utils.int_array_to_hex_str(resp[12:14]))
-        ic_ref = utils.int_array_to_hex_str(resp[14:15])
+        #parsing["MEM_SIZE_PRETTY_PRINT"] = "%d bytes : %d blocks of %d bytes (%s)" % (full_mem_size, self._block_num, block_size, utils.int_array_to_hex_str(resp[12:14]))
+        parsing["MEM_SIZE_PRETTY_PRINT"] = "%d bytes : %d blocks of %d bytes (%s)" % (full_mem_size, self._block_num, block_size, utils.int_array_to_hex_str(resp[10:11]))
+        #ic_ref = utils.int_array_to_hex_str(resp[14:15])
+        ic_ref = utils.int_array_to_hex_str(resp[11:12])
+        
         parsing["IC_REF"] = "%s (%s)" % (self.Manufacturer_codes[ic_ref], ic_ref)
 
         self._get_sys_info_resp = parsing
```

**Comment by gvinet** [comment]
Corrected in 28f73af67b5d942a880077106594bf48b7530da2 (at last!).
